postal_address/tests/test_territory.py

```diff
@@ -140,7 +146,7 @@ def test_alias_normalization(self):
 
         # Check country alias to a subdivision.
         self.assertEquals(
-            list(territory_parents_codes('TA')),
+            list(territory_parents_codes('SH-TA')),
```

Comment:
Currently it would fail but I can fix it by adding a resolution against `COUNTRY_ALIAS_TO_SUBDIVISION` in `territory_parents`. This would send the same result for both queries. Would that be ok ?